### DISCUSSION TITLE
Sections view is great, would like to configure scrolling a bit differently

### DISCUSSION THREAD

**Discussion by comdata** [home-assistant/frontend#20038] [Dashboard]
Hi,

first of sections view is great.

I would like to configure the scrolling a bit differently which might be more consistent.

So left<->right should scroll through the sections.

Up<->Down, inside the section or if you have a large section.

**Comment by noxhirsch** [comment]
While this might be nice on a touchscreen, it would be really annoying on a computer without touch.
